packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_search_bar.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC, useState } from 'react';
+import { MagnifyingGlass, X } from '@phosphor-icons/react';
+
+import { ComponentType } from './../types';
+
+import 'components/component_kit/new_designs/cw_search_bar.scss';
+import { IconName } from '../cw_icons/cw_icon_lookup';
+import { getClasses } from '../helpers';
+import { ValidationStatus } from '../cw_validation_text';
+import { CWText } from '../cw_text';
+import { CWCommunityAvatar } from '../cw_community_avatar';
+
+type BaseSearchBarProps = {
+  autoComplete?: string;
+  autoFocus?: boolean;
+  containerClassName?: string;
+  defaultValue?: string | number;
+  value?: string | number;
+  iconLeft?: IconName;
+  iconLeftonClick?: () => void;
+  inputValidationFn?: (value: string) => [ValidationStatus, string] | [];
+  label?: string | React.ReactNode;
+  maxLength?: number;
+  name?: string;
+  onInput?: (e) => void;
+  onenterkey?: (e) => void;
+  onClick?: (e) => void;
+  placeholder?: string;
+  tabIndex?: number;
+  manualStatusMessage?: string;
+  manualValidationStatus?: ValidationStatus;
+};
+
+type InputStyleProps = {
+  inputClassName?: string;
+  disabled?: boolean;
+};
+
+type InputInternalStyleProps = {
+  hasLeftIcon?: boolean;
+  isTyping?: boolean;
+};
+
+type SearchBarProps = BaseSearchBarProps &
+  InputStyleProps &
+  InputInternalStyleProps &
+  React.HTMLAttributes<HTMLDivElement>;
+
+type TagProps = {
+  communityName: string;
+  disabled?: boolean;
+  onClick: () => void;
+};
+
+const Tag: FC<TagProps> = ({ communityName, disabled, onClick }) => {
```

Comment:
this tag should be in separate file. Could be `CWTag` component

Reply:
Addressed as suggested.